src/Gamepage/Gamepage.js

```diff
@@ -0,0 +1,40 @@
+import React, {Component} from 'react';
+import './Gamepage.css';
+
+export class Gamepage extends Component {
+  constructor(props) {
+    super(props) 
+    
+    this.state = {
+      hints: [],
+      correctGuesses: [],
```

Comment:
Ooh this is clever. Since this mirrors the `game` properties exactly, when we later link the `Game` instance with the `GamePage`, it will be easy to keep them aligned.

Reply:
Yeah I figured we could use the same naming technique, as to cut down on confusion, and I totally did it on purpose, so don't even think that I accidentally did the right thing...